projects/plugins/boost/wp-js-data-sync.php

```diff
@@ -136,3 +138,10 @@ function jetpack_boost_initialize_datasync() {
 
 $entry = new Modules_Status_Entry( JETPACK_BOOST_DATASYNC_NAMESPACE, 'modules_state' );
 jetpack_boost_register_option( 'modules_state', $modules_state_schema, $entry );
+
+/**
+ * Register Minify Excludes stores.
+ */
+// @todo - add validations for these options.
```

Comment:
TODOs should be turned into tickets.

Reply:
Added to the list in https://github.com/Automattic/jetpack/issues/30080.

Reply:
Done in 36f649b1df3ab4d9617182d5f29937a02ad0f4e8.